### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Keyboard: Focus is not visible on "Ok" and "Cancel" buttons in the popup

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#90]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard; #MAS2.4.7

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.4.7

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authentication and press enter
6. Tab till windows tab and press enter
7. Turn off the first toggle button in the page
8. Tab till Ok and Cancel buttons in the popup
9. Verify focus is visible on "Ok" and "Cancel" buttons in the popup

**Actual Result:**
Focus is not visible on "Ok" and "Cancel" buttons in the popup

**Expected Result:**
Focus should visible on "Ok" and "Cancel" buttons in the popup

**Note:**
This issue has been observed in all the popups

**User Impact:**
Users with disabilities finds its difficult in working through the application if focus is not properly visible on "Ok" and "Cancel" buttons in the popup

**Attachments:**
![bug90_web server_authentication_focus visible_ok cancel](https://user-images.githubusercontent.com/42638450/44644781-f2790f80-a9f2-11e8-8d82-19d6d248cbf0.png)

**Comment by InduPriya1805** [comment]
![bug90_web server_authentication_focus visible_ok cancel](https://user-images.githubusercontent.com/42638450/44644713-ab8b1a00-a9f2-11e8-9ece-61ed6c2ef6c1.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will be removed.
